builder-base/scripts/download_golang.sh

```diff
@@ -60,5 +61,10 @@ function build::go::download(){
     done
 }
 
-build::go::download "${VERSION}" "$OUTPUT_DIR" "x86_64"
-build::go::download "${VERSION}" "$OUTPUT_DIR" "aarch64"
+if [[ $( echo $ARCHITECTURE | cut -d ',' -f1 ) == "linux/amd64" ]] ; then 
+    build::go::download "${VERSION}" "$OUTPUT_DIR" "x86_64"
+fi
+
+if [[ $( echo $ARCHITECTURE | cut -d ',' -f2 ) == "linux/arm64" ]] ; then 
```

Comment:
hm this assumes that the`linux/amd64, linux/arm64` are always in the same order right? i think that'll bite us at some point in the future, can we do this with a 'find' or some other thing that doesn't care about the order?